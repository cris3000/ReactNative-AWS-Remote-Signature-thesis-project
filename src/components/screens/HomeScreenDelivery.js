import React from 'react'
import {
  StyleSheet,
  View,
  Text,
} from 'react-native'
import Ionicons from 'react-native-ionicons'


//class for HomeScreen For DeliveryUser
export default class HomeScreenDelivery extends React.Component {

  render() {
    return (
    
      
      <View style={styles.container}> 
      <Text style={styles.textStylex}>Icon Definitions</Text>
      <Text style={styles.textStylexx}><Ionicons style={styles.IconStyle} name="ios-home" />   Home 
      {' '}  <Ionicons  style={styles.IconStyle} name="ios-settings" /> Add Delivery to Database</Text>
    
      <Text style={styles.textStylexx}>  <Ionicons style={styles.IconStyle} name="ios-car" /> {' '} Available Deliveries  
      {' '}  <Ionicons  style={styles.IconStyle} name="ios-checkbox" /> Delivery Response   </Text>
      <Text style={styles.textStylexx}>  <Ionicons  style={styles.IconStyle}  name="ios-map" />   Map   </Text>
        <Testapp/>
    
        <Text style={styles.textStyle}>Instructions </Text>
        <Text style={styles.textStylex}>Check Delivery List and Submit Request on the necessary Delivery</Text>
        <Text style={styles.textStylex}>Add Delivery to database by Inputting Delivery Details on Delivery Page</Text>
        <Text style={styles.textStylex}>Check Response Page for the Customers Reponse</Text>
        <Text> </Text>
      </View>

    )
  }
}
//Styles
const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#5059ae',
    alignItems: 'center',
    justifyContent: 'center',
  },
  textStyle: {
    fontWeight: 'bold',
    fontSize: 25,
    padding: 10,
    color: '#fff'
  },
  IconStyle:{
    color: '#2fb007',
  },
  textStylex: {
    fontWeight: 'bold',
    fontSize: 18,
    padding: 10,
    color: '#fff'
  },
  textStylexx: {
    fontWeight: 'bold',
    fontSize: 12,
    padding: 0,
  
    flexDirection: 'column',
    color: '#fff'
  }
})