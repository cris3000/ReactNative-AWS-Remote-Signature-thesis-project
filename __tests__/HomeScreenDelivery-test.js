// __tests__/HomeScreen-page-test.js
import 'react-native';
import React from 'react';
import HomeScreenDelivery from '../src/components/screens/HomeScreenDelivery';

import renderer from 'react-test-renderer';
// Testing whether screen renders
test('renders correctly', () => {
  const tree = renderer.create(
   <HomeScreenDelivery />
    ).toJSON();
  expect(tree).toMatchSnapshot();
});