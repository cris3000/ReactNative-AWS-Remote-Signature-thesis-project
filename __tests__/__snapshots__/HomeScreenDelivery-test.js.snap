// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#5059ae",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontSize": 18,
        "fontWeight": "bold",
        "padding": 10,
      }
    }
  >
    Icon Definitions
  </Text>
  <Text
    style={
      Object {
        "color": "#fff",
        "flexDirection": "column",
        "fontSize": 12,
        "fontWeight": "bold",
        "padding": 0,
      }
    }
  >
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "fontFamily": "Ionicons",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {
            "color": undefined,
            "fontSize": 30,
          },
          Object {
            "color": "#2fb007",
          },
        ]
      }
    >
      
    </Text>
       Home
     
      
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "fontFamily": "Ionicons",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {
            "color": undefined,
            "fontSize": 30,
          },
          Object {
            "color": "#2fb007",
          },
        ]
      }
    >
      
    </Text>
     Add Delivery to Database
  </Text>
  <Text
    style={
      Object {
        "color": "#fff",
        "flexDirection": "column",
        "fontSize": 12,
        "fontWeight": "bold",
        "padding": 0,
      }
    }
  >
      
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "fontFamily": "Ionicons",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {
            "color": undefined,
            "fontSize": 30,
          },
          Object {
            "color": "#2fb007",
          },
        ]
      }
    >
      
    </Text>
     
     
     Available Deliveries
     
      
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "fontFamily": "Ionicons",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {
            "color": undefined,
            "fontSize": 30,
          },
          Object {
            "color": "#2fb007",
          },
        ]
      }
    >
      
    </Text>
     Delivery Response   
  </Text>
  <Text
    style={
      Object {
        "color": "#fff",
        "flexDirection": "column",
        "fontSize": 12,
        "fontWeight": "bold",
        "padding": 0,
      }
    }
  >
      
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "fontFamily": "Ionicons",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {
            "color": undefined,
            "fontSize": 30,
          },
          Object {
            "color": "#2fb007",
          },
        ]
      }
    >
      
    </Text>
       Map   
  </Text>
  <View>
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    />
  </View>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontSize": 25,
        "fontWeight": "bold",
        "padding": 10,
      }
    }
  >
    Instructions 
  </Text>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontSize": 18,
        "fontWeight": "bold",
        "padding": 10,
      }
    }
  >
    Check Delivery List and Submit Request on the necessary Delivery
  </Text>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontSize": 18,
        "fontWeight": "bold",
        "padding": 10,
      }
    }
  >
    Add Delivery to database by Inputting Delivery Details on Delivery Page
  </Text>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontSize": 18,
        "fontWeight": "bold",
        "padding": 10,
      }
    }
  >
    Check Response Page for the Customers Reponse
  </Text>
  <Text>
     
  </Text>
</View>
`;
